### DISCUSSION TITLE
Primereact Tooltip props `children` and `content` have different types

### DISCUSSION THREAD

**Discussion by lukaszsobek** [primefaces/community#527] [PrimeReact] [answered]
When looking at the documentation `children` and `content` serve the same purpose:

<img width="800" alt="image" src="https://github.com/primefaces/community/assets/23530054/6ea135eb-4f7c-4822-9f50-27de398da82b">

but their types are different:

<img width="800" alt="image" src="https://github.com/primefaces/community/assets/23530054/0664c043-dfd5-4f50-b68b-6c222809829f">

which leads me to the question:
which of the above needs to be aligned - the description or the types?

**Comment by melloware** [comment]
This looks like a bug.  Let me create an issue.

**Comment by melloware** [comment, marked as answer]
Fixed for 10.2.0: https://github.com/primefaces/primereact/issues/5376

I updated the type of `content` to also be ReactNode but both content and children have slightly different use cases.

The `content` can be used in the Reactive example like this: https://primereact.org/tooltip/#reactive

**Reply by lukaszsobek** [reply]
Awesome, thank you very much for the quick response and fix :)
